site/conf.py

```diff
@@ -20,7 +20,7 @@
 BLOG_TITLE = "Bitácora de vuelo"  # (translatable)
 # This is the main URL for your site. It will be used
 # in a prominent link. Don't forget the protocol (http/https)!
-SITE_URL = "http://www.taniquetil.com.ar/blog/"
+SITE_URL = "http://beta.taniquetil.com.ar/"
```

Comment:
Why this change? What has to do with the tag cloud?

Reply:
yes. 
It fails using subdirs. This is why I asked you about the definite configuration on server.